ubid.rb

```diff
@@ -32,16 +32,14 @@ class UBID
 
   # types
   TYPE_VM = "vm"
-  TYPE_STORAGE_VOLUME = "v1"
+  TYPE_VM_STORAGE_VOLUME = "v1"
```

Comment:
@pykello, do we use v1 instead of vl intentionally or is it typo?

Reply:
I saw your PR https://github.com/ubicloud/ubicloud/pull/329/files and understand that this is intentional.